### PR TITLE
Fixes loot crates and animal attacks

### PR DESCRIPTION
**Pull request by Razharas**
Fixes loot crates and animal attacks
Fixes https://github.com/tgstation/-tg-station/issues/10412

### CODE REVIEW

**Comment by Zilenan91** [comment]
I thought this was intended.

**Comment by tkdrg** [comment]
I wouldn't make them monsterproof barricades
Just make them delete contents instead IMO

**Comment by phil235** [comment]
What tkdrg said, the crate has a content-destruction mechanism, can't open it with anything but the right combination (no explosion, no animal attack, etc..)

File: code/modules/mining/abandoned_crates.dm [hunk 1]
```diff
@@ -179,6 +179,9 @@
 	else
 		return ..()
 
+/obj/structure/closet/crate/secure/loot/attack_animal()
+	qdel(src)
+
```
Comment (phil235) [inline code/modules/mining/abandoned_crates.dm]: Doesn't this mean that a corgi nuzzling the crate will delete it?

**Comment by tkdrg** [comment]
Yeah. This pr still has outstanding issues and frankly I'm disappointed that remie merged it as is despite the fact that two maintainers requested changes.

**Comment by RemieRichards** [comment]
@tkdrg Raz fulfilled your contract by making it delete itself and it's contents, I *DID* tell him to do the things you two asked, and he replied with that.

So bite me.

**Comment by tkdrg** [comment]
Yes ok but he did it wrong

**Pull request merged**